scripts/cargo.sh

```diff
@@ -1,14 +1,27 @@
 #!/bin/sh
 
+# $1 -> @CURRENT_SOURCE_DIR@ as replaced by meson, we cd into it
+# $2 -> Where to place the finished binaries, meson build_root with @OUTDIR@
```

Comment:
There's a point missing between meson and build_root

Reply:
fixed